### DISCUSSION TITLE
2010/09/24/%E6%B0%B4%E7%97%98

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#297] [Announcements]
# 2010/09/24/%E6%B0%B4%E7%97%98

老婆怀孕之后，我们家就开始到处漏水。

https://blog.qizhen.xyz/2010/09/24/%E6%B0%B4%E7%97%98

**Comment by ruanqizhen** [comment]
Jucao
2010年09月26日 12:01 编辑
恭喜了，Qizhen，要升级了！
